README.md

```diff
@@ -56,32 +56,19 @@ Example of `package.json` with good meta data:
 }
 ```
 
-## 📦 Install
-
-With npm:
-
-```sh
-npm i -g readme-md-generator
-```
-
-With yarn:
-
-```sh
-yarn global add readme-md-generator
-```
-
 ## 🚀 Usage
 
 Just run the following command at the root of your project and answer questions:
```

Comment:
```suggestion
Make sure you have [npx](https://www.npmjs.com/package/npx) installed (`npx` is shipped by default since npm `5.2.0`)

Just run the following command at the root of your project and answer questions:
```